packages/orchestrator/main.go

```diff
@@ -287,16 +286,12 @@ func run(config cfg.Config) (success bool) {
 		}})
 	}
 
-	// pubsub
-	var redisPubSub pubsub.PubSub[event.SandboxEvent, struct{}]
+	// Redis sandbox events delivery target
 	if redisClient != nil {
-		redisPubSub = pubsub.NewRedisPubSub[event.SandboxEvent, struct{}](redisClient, "sandbox-webhooks")
-	} else {
-		redisPubSub = pubsub.NewMockPubSub[event.SandboxEvent, struct{}]()
+		sbxEventsDeliveryRedis := event.NewRedisStreamsDelivery[event.SandboxEvent](redisClient, event.SandboxEventsStreamName)
```

Comment:
Lets sync if for this PR we should stick with Redis Pub/Sub or already us Redis Streams as default.